pcsx2/R3000A.cpp

```diff
@@ -63,7 +63,8 @@ void psxReset()
 	g_iopNextEventCycle = psxRegs.cycle + 4;
 
 	psxHwReset();
-	PSXCLK = 36864000;
+	PSXCLK =  33868800;
```

Comment:
Doesn't this change mean that the IOP runs too slowly when playing PS2 games?

Reply:
This is specifically for ps1mode in CDVD.cpp the ps1mode reset this is specifically called

Reply:
I'm pretty sure that this'll affect both PS2 and PSX mode.

Reply:
I just tested it with rachet and clank. It doesn't

Reply:
Set a breakpoint here in the debugger and see it if triggers when you run a PS2 game. Because it does on my end (even boot to BIOS triggers this code).

Reply:
psxReset gets called when the PS2 side resets, it was 36Mhz for a reason.